templates/components/pager.html.twig

```diff
@@ -39,7 +39,9 @@
 {% endif %}
 
 {% set href = href|default('') %}
-{% set start = start|default(1) %}
+
+{# "start" is 0 based, the 1 based index is "current_start" #}
+{% set start = start|default(0) %}
```

Comment:
Sounds like a logic that should not be handled template side, no? 
But maybe it's complicated to do it elsewhere (therefore this is not a blocker).